feed/history/boost_1_73_0.qbk

```diff
@@ -71,6 +71,11 @@ Please keep the list of libraries sorted in lexicographical order.
     construction for the legacy version of libstdc++-v3 shipped with GCC 4.8
     (which can also be used by Clang).
 
+* [phrase library..[@/tools/build/ B2]:]
```

Comment:
This is currently under "Updated Libraries". Can you create a new heading "Updated Tools" (below Updated Libraries) and place this beneath that?